### PR TITLE
v0.12.0 updates

### PR DESCRIPTION
**Pull request by nlinx**
Adds a release notes doc for v0.12.0 and updates the version in the README.md to 0.12.0.

### CODE REVIEW

File: releases-notes/v0.12.0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# v0.12.0
+
+## Notable Changes
+
+1. Allow operators to override the ingress-nginx-controller name
+2. Introduce Space scoped labels that can be applied via mxe install
```
Comment (AlainRoy) [inline releases-notes/v0.12.0.md]: I don't see documentation on either of these. Do need to describe how they can be used?

File: README.md [hunk 2]
```diff
@@ -74,22 +79,24 @@ tokens you have received.
    that you can add a public DNS record for `kubectl` requests to find the router,
    hence the control plane instance.
 
-   `CLUSTER_TYPE` enables you to deploy cluster specific resources during 
+   `CLUSTER_TYPE` enables you to deploy cluster specific resources during
    installation. Currently the only supported types are `kind` or `aks`. `eks`
    and `gke` will be supported in a future release.
```
Comment (tnthornton) [inline README.md]: Could we update this line to add `gke` to the list with `kind` and `aks`? It is supported and confirmed to work as of this moment in time.

File: releases-notes/v0.12.0.md (hunk 1 above)
Comment (tnthornton) [inline releases-notes/v0.12.0.md]: Some nits:

```suggestion
1. Allow operators to override the ingress-nginx-controller name. Operators can accomplish this by setting the following property when installing the `mxp` chart: 
`mcp-controller.mcp.hostCluster.ingressNginx.controllerName=mycontrollername`

2. Introduce Space scoped labels that can be applied via mxp install. Operators can accomplish this by setting the following property when installing the chart:
`global.labels.key=value`
```

**Review by tnthornton — COMMENTED** [review]
Looks good @nlinx ! I have a few comments below. Let me know if you'd like any clarifications 👍.

File: releases-notes/v0.12.0.md (hunk 1 above)
Comment (nlinx) [reply releases-notes/v0.12.0.md]: updated!

File: README.md (hunk 2 above)
Comment (nlinx) [reply README.md]: added

File: releases-notes/v0.12.0.md (hunk 1 above)
Comment (nlinx) [reply releases-notes/v0.12.0.md]: updated! thanks!

**Review by tnthornton — APPROVED** [review]

**Review by branden — APPROVED** [review]

**Review by AlainRoy — APPROVED** [review]

**Pull request merged**